sylius/refund-plugin/0.4/config/packages/sylius_refund.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: "@SyliusRefundPlugin/Resources/config/app/config.yml" }
```

Comment:
Out of curiosity, why is it not imported automatically by the bundle?

Reply:
Do you mean by loading it in a DI extension?

Reply:
yes, that :)

Reply:
This is a file containing the configuration for other bundles, like `sylius_grid`, `sylius_resource` or `winzou_state_machine` etc.

Loading it in extension's `load` method would not work as it does not have access to other bundles. Loading it in extension's `prepend` method would work, but then we had an issue with end-user trying to add a different config for another bundle having `disallowNewKeysInSubsequentConfigs` in configuration or no way to delete / reorder a collection of items in config. 

Importing the bundle's config file explicitly in the end-user config makes it possible to copy that config and customise it to one's need, while still allowing for config changes in the new releases. Also, it uses YAML which everyone is familiar with instead of arrays in PHP in DI extension, but that's only a side-effect.